src/decoders/mrw.rs

```diff
@@ -46,7 +46,7 @@ impl<'a> MrwDecoder<'a> {
           }
         }
         0x545457 => { // TTW
-          // Base value for offsets needs to be at the beginning of the 
```

Comment:
Whoops, it seems like I forgot to turn off my editor's trim whitespace on save feature.